src/libmongoc/src/mongoc/mongoc-cmd.c

```diff
@@ -626,6 +626,7 @@ _mongoc_cmd_parts_assemble_mongod (mongoc_cmd_parts_t *parts,
          break;
       case MONGOC_TOPOLOGY_SHARDED:
       case MONGOC_TOPOLOGY_UNKNOWN:
+      case MONGOC_TOPOLOGY_LOAD_BALANCED:
```

Comment:
This PR only covers updates to sessions behavior. But adding this case was necessary to avoid compiler warnings for a missing case on an enum value.